src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -857,9 +858,36 @@ public void canCheckOutInProcessItemWithRequest() {
     assertThat(smallAngryPlanet, hasItemStatus(CHECKED_OUT));
   }
 
+  @Test
+  public void cannotCheckOutWhenItemIsNotLoanable() {
+    IndividualResource notLoanablePolicy = loanPoliciesFixture.create(
+      new LoanPolicyBuilder()
+        .withName("Not Loanable Policy")
+        .withLoanable(false));
+
+    useFallbackPolicies(
+      notLoanablePolicy.getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.inactiveNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId(),
+      lostItemFeePoliciesFixture.facultyStandard().getId());
+
+    InventoryItemResource nod = itemsFixture.basedUponNod();
+    IndividualResource steve = usersFixture.steve();
+    Response response = loansFixture.attemptCheckOutByBarcode(nod, steve);
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Item is not loanable"),
+      hasItemBarcodeParameter(nod),
+      hasLoanPolicyParameters(notLoanablePolicy))));
+  }
+
   @Test
   public void checkOutFailsWhenCirculationRulesReferenceInvalidLoanPolicyId() {
-    setInvalidLoanPolicyReferenceInRules("some-loan-policy");
+    UUID invalidLoanPolicyId = UUID.randomUUID();
+    loanPoliciesFixture.create(invalidLoanPolicyId);
+    setInvalidLoanPolicyReferenceInRules(invalidLoanPolicyId.toString());
+    loanPoliciesFixture.cleanUp();
```

Comment:
Can only the specific loan policy be deleted, rather than all of them? I think that would explicitly describe this scenario